package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "Sith",
+  "version": "3",
+  "description": "Le web Sith de l'AE",
+  "main": "index.js",
+  "scripts": {
+    "compile": "webpack --mode production",
+    "compile-dev": "webpack --mode development",
+    "serve": "webpack --mode development --watch"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "GPL-3.0-only",
+  "devDependencies": {
+    "webpack": "^5.94.0",
+    "webpack-cli": "^5.1.4"
+  },
+  "sideEffects": [
+    ".css"
+  ],
+  "dependencies": {
+    "@babel/core": "^7.25.2",
+    "@babel/preset-env": "^7.25.4",
+    "alpinejs": "^3.14.1",
+    "babel-loader": "^9.2.1",
+    "css-loader": "^7.1.2",
+    "css-minimizer-webpack-plugin": "^7.0.0",
+    "easymde": "^2.18.0",
+    "expose-loader": "^5.0.0",
+    "glob": "^11.0.0",
+    "jquery": "^3.7.1",
+    "jquery-ui": "^1.14.0",
+    "jquery.shorten": "^1.0.0",
+    "mini-css-extract-plugin": "^2.9.1",
+    "source-map-loader": "^5.0.0"
+  }
```

```suggestion
  "devDependencies": {
    "webpack": "^5.94.0",
    "webpack-cli": "^5.1.4",
    "babel-loader": "^9.2.1",
    "css-loader": "^7.1.2",
    "css-minimizer-webpack-plugin": "^7.0.0",
    "@babel/core": "^7.25.2",
    "@babel/preset-env": "^7.25.4",
    "mini-css-extract-plugin": "^2.9.1",
    "expose-loader": "^5.0.0",
  },
  "sideEffects": [
    ".css"
  ],
  "dependencies": {
    "alpinejs": "^3.14.1",
    "easymde": "^2.18.0",
    "glob": "^11.0.0",
    "jquery": "^3.7.1",
    "jquery-ui": "^1.14.0",
    "jquery.shorten": "^1.0.0",
    "source-map-loader": "^5.0.0"
  }
```

Aussi, pourquoi il y a toutes ses dépendances JS, alors que presque tout le style du site est géré par `libsass` ?

---

Parce que c'est pour récupérer le js packagé dans les paquets npm
C'est vraiment pas pour gérer notre css

---

En gros, t'as css-loader pour gérer les imports de fichiers css dans les .js (genre quand tu requiret les css de jquery-ui, easymde ou bientôt fontawesome) et t'as css-minimizer-webpack-plugin pour les minifier parce que sinon ils sortent du workflow des scss de django malhereusement.

Par contre, au vu de la configuration, ils sont quand même gérés avec les manifests donc ça c'est cool :)